iree/compiler/Conversion/LinalgToSPIRV/KernelDispatchUtils.h

```diff
@@ -61,6 +70,57 @@ LogicalResult createNumWorkgroupsFromLinearizedResultShape(
 /// workgroups to use at launch time.
 FuncOp getNumWorkgroupsFn(FuncOp entryPointFn);
 
+using TileSizesListType = SmallVector<SmallVector<int64_t, 4>, 1>;
+/// Based on the linalg operations in a dispatch region, the number of levels of
```

Comment:
... the tile sizes, the workgroup size, etc.. We call these parameters together as the "launch configuration". This class...